### PR TITLE
Markdown updates for 3.2.0

### PR DESCRIPTION
**Pull request by ralfhandl**
* 3.2.0 port of
  * #3991
* Run markdownlint after mdv on 3.1.1.md only because the older files won't pass
  * Part of #3596

### CODE REVIEW

**Review by mikekistler — APPROVED** [review]
Looks good! 👍

**Review by handrews — CHANGES_REQUESTED** [review]
I'm confused as to what happens in which PR, so I'm just going to say the same thing on this one:


@ralfhandl can you re-do this with the tables not expanded since we agreed on that for the other branches? That will keep the change history so much less polluted on this branch.

Ideally I'd prefer not collapsing the spaces after periods, but I'm not going to my heels in on it (but it does create more noise and makes it harder to find sentence changes in a fixed-width editor).

**Comment by ralfhandl** [comment]
The "table condenser" is currently only in #4040. @mikekistler has approved it half a dozen times already, and I'd happily merge it and port it to 3.1.1 and 3.2.0 with a second approval.

**Comment by ralfhandl** [comment]
This is the first part of several PRs to sync 3.2.0 with 3.1.1
* #4025 - remove TOC, prettier, markdownlint
* #4026 - stylistic harmonization
* #4035 - property, field, keyword cleanup

We can merge them one by one, thus minimizing the differences, or just merge the last one with the cumulative changes and close the others.

We'll need an additional diff to 3.1.1 to add the changes we did since the last of these PRs was created.

File: versions/3.2.0.md [hunk 1]
```diff
@@ -1183,76 +1124,76 @@ There are four possible parameter locations specified by the `in` field:
...
 
-See [Appendix E](#percentEncodingAndFormMediaTypes) for a discussion of percent-encoding, including when delimiters need to be percent-encoded and options for handling collisions with percent-encoded data.
+See [Appendix E](#appendix-e-percent-encoding-and-form-media-types) for a discussion of percent-encoding, including when delimiters need to be percent-encoded and options for handling collisions with percent-encoded data.
 
 ##### Style Examples
 
 Assume a parameter named `color` has one of the following values:
 
-```
+```js
```
Comment (handrews) [inline versions/3.2.0.md:1194]: This isn't exactly accurate but looks nicer than plain text so OK :-)

**Review by handrews — APPROVED** [review]

**Comment by ralfhandl** [comment]
Merging as @mikekistler has approved before table condensing.

**Pull request merged**